stdlib/public/BackDeployConcurrency/CompatibilityOverrideConcurrency.def

```diff
@@ -278,6 +278,10 @@ OVERRIDE_TASK_GROUP(taskGroup_addPending, bool,
                     swift::, (TaskGroup *group, bool unconditionally),
                     (group, unconditionally))
 
+OVERRIDE_TASK_GROUP(task_cancel_group_child_tasks, void,
```

Comment:
Is it safe to re-order these? There's a big warning at the top of the file suggesting that "Moving or removing entries in this file will break the ABI".